File: code-samples/video/meetings.java
package CreateMeeting;
import com.ringcentral.*;
import java.io.IOException;

public class CreateMeeting {
  static RestClient restClient;

  public static void main(String[] args) {
    var obj = new CreateMeeting();
    try {
      // Instantiate the SDK
      restClient = new RestClient(System.getenv("RC_APP_CLIENT_ID"), System.getenv("RC_APP_CLIENT_SECRET"), System.getenv("RC_SERVER_URL"));

      // Authenticate a user using a personal JWT token
      restClient.authorize(System.getenv("RC_USER_JWT"));

      obj.create_meeting();
    } catch (RestException | IOException e) {
      e.printStackTrace();
    }
  }

  /*
  * Create an instant RCV meeting
  */
  public void create_meeting() throws RestException, IOException{
    var bodyParams = new CreateBridgeRequest();
    bodyParams.name = "Test Meeting";
    bodyParams.type = "Instant";
    var response = restClient.rcvideo().v2().account("~").extension("~").bridges().post(bodyParams);
    System.out.println("Start Your Meeting: " + response.discovery.web);
  }
}

File: code-samples/video/meetings.cs
using System;
using RingCentral;

namespace CreateMeeting
{
  class Program
  {
    static RestClientrest Client;
    static async Task Main()
    {
      try
      {
        DotEnv.Load();
        // Instantiate the SDK
        restClient = new RestClient(Environment.GetEnvironmentVariable("RC_CLIENT_ID"),
                                    Environment.GetEnvironmentVariable("RC_CLIENT_SECRET"),
                                    Environment.GetEnvironmentVariable("RC_SERVER_URL"));

        // Authenticate a user using a personal JWT token
        await restClient.Authorize(Environment.GetEnvironmentVariable("RC_JWT"));
        await create_meeting();
      }
      catch (Exceptione e)
      {
        Console.WriteLine(e.Message);
      }
    }
    /*
    Create a an instant RCV meeting
    */
    private static async Task create_meeting()
    {
      try
      {
        var bodyParams = new CreateBridgeRequest { name = "Test meeting", type = "Instant" };
        var response = await restClient.Rcvideo().V2().Account("~").Extension("~").Bridges().Post(bodyParams);
        Console.WriteLine("Start Your Meeting:  " + response.discovery.web);
      }
      catch (Exception e)
      {
        Console.WriteLine(e.Message);
      }
    }
  }
}
